app/src/main/java/com/hedvig/app/feature/table/TableAdapter.kt

```diff
@@ -19,11 +19,11 @@ class TableAdapter : ListAdapter<TableAdapter.UpcomingAgreementItem, TableAdapte
 
     fun setTable(table: Table) {
         submitList(
-            listOf(
```

Comment:
A pattern I've adopted when the declarative approach becomes a bit messy is to simply create an `ArrayList` and use `if`s to conditionally `add` things to the final list. Could be something to consider here if you agree 😄